### PR TITLE
Allow this bundle to work with symfony3

### PR DESCRIPTION
**Pull request by benji07**

### CODE REVIEW

File: Form/Extension/ChoiceTypeExtension.php [hunk 1]
```diff
@@ -25,7 +27,7 @@ class ChoiceTypeExtension extends AbstractTypeExtension
      */
     public function getExtendedType()
     {
-        return 'choice';
+        return method_exists(AbstractType::class, 'getBlockPrefix') ? ChoiceType::class: 'choice';
```
Comment (Tom32i) [inline Form/Extension/ChoiceTypeExtension.php]: Missing space after "ChoiceType::class"

File: Form/Extension/ChoiceTypeExtension.php [hunk 2]
```diff
@@ -35,7 +37,11 @@ public function finishView(FormView $view, FormInterface $form, array $options)
     {
         foreach ($view as $child) {
             $child->vars['block_prefixes'][] = 'choice_item';
-            $child->vars['block_prefixes'][] = $form->getName() . '_item';
+
+            $blockPrefix = $form->getName() . '_item';
+            if (!in_array($blockPrefix, $child->vars['block_prefixes'])) {
```
Comment (Tom32i) [inline Form/Extension/ChoiceTypeExtension.php]: :+1:

File: Form/Extension/CollectionTypeExtension.php [hunk 3]
```diff
@@ -27,7 +29,7 @@ class CollectionTypeExtension extends AbstractTypeExtension
      */
     public function getExtendedType()
     {
-        return 'collection';
+        return method_exists(AbstractType::class, 'getBlockPrefix') ? CollectionType::class: 'collection';
```
Comment (Tom32i) [inline Form/Extension/CollectionTypeExtension.php]: We should have two branch! Not a bundle that check the Symfony version in an `if` statement.

**Comment by maximecolin** [comment]
We must fix Twig deprecated too :
* Twig Test "sameas" is deprecated. Use "same as" instead in @ElaoForm/Form/form_elao_layout.html.twig
* Global variable "_self" is deprecated

**Pull request merged**